dask/array/fft.py

```diff
@@ -10,6 +11,9 @@
 except ImportError:
     scipy = None
 
+from dask.array.creation import linspace as _linspace
+from dask.array.core import concatenate as _concatenate
```

Comment:
Ok. Would it just be better to just import `dask.array` and use these functions from there?